app/components/graph-editor/README.md

```diff
@@ -1,6 +1,98 @@
-_Last updated: January 31, 2017_
-
-# Notes
+_Last updated: February 11, 2017_
+
+# Remarks
+The graph editor component is a custom Angular2 component that is intended to be
```

Comment:
I think the sentiment is it should be decoupled such that we could move it to its own package (and it is). But that currently moving it wouldn't provide a tangible benefit towards completing Beta.